src/main/java/Interface/InterfacePrintInformation.java

```diff
@@ -0,0 +1,9 @@
+package Interface;
+
+import models.User;
+
+public interface InterfacePrintInformation {
```

Comment:
it is enough to name it just "Printable". We usually use adjectives for interfaces